tutorials/tts/2_Inference_DurationPitchControl.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TTS Inference Prosody Control\n",
+    "\n",
+    "This notebook is intended to teach users how to control duration and pitch with the FastPitch model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# License\n",
+    "\n",
+    "> Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.\n",
+    ">\n",
+    "> Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "> you may not use this file except in compliance with the License.\n",
+    "> You may obtain a copy of the License at\n",
+    ">\n",
+    ">     http://www.apache.org/licenses/LICENSE-2.0\n",
+    ">\n",
+    "> Unless required by applicable law or agreed to in writing, software\n",
+    "> distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "> See the License for the specific language governing permissions and\n",
+    "> limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# # If you're using Google Colab and not running locally, uncomment and run this cell.\n",
+    "# !apt-get install sox libsndfile1 ffmpeg\n",
+    "# !pip install wget unidecode\n",
+    "# BRANCH = 'main'\n",
+    "# !python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[tts]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports\n",
```

Comment:
"Imports" -> "Setup" + update description? Since there are some helper functions and model downloads as well.

Reply:
done